src/Network/Network/AzureFirewallPolicy/ApplicationRuleCondition/NewAzureFirewallPolicyApplicationRuleCommand.cs

```diff
@@ -70,6 +70,12 @@ public override void Execute()
         {
             base.Execute();
 
+            // One of SourceAddress or SourceIpGroup must be present
```

Comment:
can you also add same parameters for this condition?

Reply:
Done